### PR TITLE
✨ pullquote redesign

### PR DESCRIPTION
**Pull request by ikesau**
## Context

Part of https://github.com/owid/owid-grapher/issues/4851

[Figma](https://www.figma.com/design/8XR11jiphFH5YwegxkhqxE/Archie-Components-Revamp?node-id=1-3173&p=f&t=wXO7JTTRi3ikq3jW-0)

To get these to position correctly, we have to use the CSS `float` attribute (so that the body text wraps around the pullquote) but `float` doesn't work when in a grid container. Thus we have to create a new div context that isn't a grid container that we can position the pull-quote inside using float.

e.g. instead of

```html
<div class="grid">
  <blockquote>A quote</blockquote>
  <p>I am text that the pullquote is going to intrude into</p>
</div>
```

we have:
```html
<div class="grid">
  <div>
    <blockquote>A quote</blockquote>
    <p>I am text that the pullquote is going to intrude into</p>
  </div>
</div>
```

Originally I had written a migration that ported the old enriched pull-quote blocks to the new shape, but they both looked pretty bad when I rebaked the site. I decided instead to have a simple migration that filters them out and then immediately republish the 2 articles with the new archie, placed appropriately.

## Screenshots / Videos / Diagrams

[Desktop](https://github.com/user-attachments/assets/a34b6eee-e579-4196-aeb7-b0d6d1c5de04)

[Mobile](https://github.com/user-attachments/assets/32d1c7de-9717-4dc2-a3a7-eccc5402af8e)

## Testing guidance

You can use the following archie:

```
{.pull-quote}
quote: I am a left-aligned quote that should span multiple lines.
align: left
[.+content]
Suspendisse commodo turpis nunc, sit amet cursus odio porttitor scelerisque. Ut vel vehicula mauris. Suspendisse maximus ut enim gravida luctus. Nunc aliquam mi et viverra gravida. Etiam ac volutpat urna. Nam nisl nisi, malesuada eget lorem non, posuere mollis mauris. Pellentesque tristique erat ut turpis tristique, condimentum faucibus metus mattis. Donec non erat justo. Vivamus velit nulla, iaculis vel metus et, tincidunt pharetra justo. Quisque lacinia fringilla odio, vitae semper urna semper quis. Sed aliquam ipsum quam, in suscipit purus fringilla et. In congue commodo cursus. 
[]
{}

Cras nec est eu justo pharetra blandit at at tellus. Etiam enim metus, aliquet ac lorem id, auctor ullamcorper diam.

{.pull-quote}
quote: I am a left-center aligned quote that should span multiple lines.
align: left-center
[.+content]
Suspendisse commodo turpis nunc, sit amet cursus odio porttitor scelerisque. Ut vel vehicula mauris. Suspendisse maximus ut enim gravida luctus. Nunc aliquam mi et viverra gravida. Etiam ac volutpat urna. Nam nisl nisi, malesuada eget lorem non, posuere mollis mauris. Pellentesque tristique erat ut turpis tristique, condimentum faucibus metus mattis. Donec non erat justo. Vivamus velit nulla, iaculis vel metus et, tincidunt pharetra justo. Quisque lacinia fringilla odio, vitae semper urna semper quis. Sed aliquam ipsum quam, in suscipit purus fringilla et. In congue commodo cursus. 
[]
{}

{.pull-quote}
quote: I am a right-center aligned quote that should span multiple lines.
align: right-center
[.+content]
Suspendisse commodo turpis nunc, sit amet cursus odio porttitor scelerisque. Ut vel vehicula mauris. Suspendisse maximus ut enim gravida luctus. Nunc aliquam mi et viverra gravida. Etiam ac volutpat urna. Nam nisl nisi, malesuada eget lorem non, posuere mollis mauris. Pellentesque tristique erat ut turpis tristique, condimentum faucibus metus mattis. Donec non erat justo. Vivamus velit nulla, iaculis vel metus et, tincidunt pharetra justo. Quisque lacinia fringilla odio, vitae semper urna semper quis. Sed aliquam ipsum quam, in suscipit purus fringilla et. In congue commodo cursus. 
[]
{}

{.pull-quote}
quote: I am a right-aligned quote that should span multiple lines.
align: right
[.+content]
Suspendisse commodo turpis nunc, sit amet cursus odio porttitor scelerisque. Ut vel vehicula mauris. Suspendisse maximus ut enim gravida luctus. Nunc aliquam mi et viverra gravida. Etiam ac volutpat urna. Nam nisl nisi, malesuada eget lorem non, posuere mollis mauris. Pellentesque tristique erat ut turpis tristique, condimentum faucibus metus mattis. Donec non erat justo. Vivamus velit nulla, iaculis vel metus et, tincidunt pharetra justo. Quisque lacinia fringilla odio, vitae semper urna semper quis. Sed aliquam ipsum quam, in suscipit purus fringilla et. In congue commodo cursus. 
[]
{}
```

Test different pullquote lengths and at all breakpoints and make sure it looks OK. There will definitely be edge cases where the browser's text-wrapping algorithm makes it look bad, but we'll just have to make sure authors know that they need to use their aesthetic judgment here.

- [ ] Does the staging experience have sign-off from product stakeholders?

**Reminder to annotate the PR diff with design notes, alternatives you considered, and any other helpful context.**

## Checklist

(delete all that do not apply)

### Before merging

- [x] Changes to CSS/HTML were checked on Desktop and Mobile Safari at all three breakpoints
- [x] Changes to HTML were checked for accessibility concerns

If DB migrations exists:
- [x] The DB type definitions have been updated

### CODE REVIEW

File: site/gdocs/components/PullQuote.scss [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+            text-align: left;
+            &::before {
+                right: unset;
+                left: 0;
+            }
+            &::after {
+                right: unset;
+                left: 0;
+            }
+        }
```
Comment (mlbrgl) [inline site/gdocs/components/PullQuote.scss:111]: nit: could be condensed

**Review by mlbrgl — APPROVED** [review]
nicely done! Just a couple of stylistic nits but nothing else came up during testing 👍
